explorer/src/context/main.tsx

```diff
@@ -168,8 +169,14 @@ export const MainContextProvider: FCWithChildren = ({ children }) => {
   const fetchServiceProviders = async () => {
     setServiceProviders({ data: undefined, isLoading: true });
     try {
-      const [res] = await Api.fetchServiceProviders();
-      setServiceProviders({ data: res, isLoading: false });
+      const res = await Api.fetchServiceProviders();
+      const resWithRoutingScorePercentage = res.map((item) => ({
+        ...item,
+        routing_score: item.routing_score
+          ? `${toPercentIntegerString(item.routing_score.toString())}%`
+          : item.routing_score,
```

Comment:
Did you miss using a default value when `item.routing_score` is undefined?

Reply:
It's OK, I set it up this way. Routing score will be a value or null. If it's null the UI will display a "-".